msbuild/Xamarin.MacDev.Tasks.Core/Tasks/CompileAppManifestTaskBase.cs

```diff
@@ -285,6 +287,13 @@ bool SetMinimumOSVersion (PDictionary plist)
 			return true;
 		}
 
+		protected string GetMinimumOSVersion (PDictionary plist, out Version version)
+		{
+			var rv = plist.Get<PString> (PlatformFrameworkHelper.GetMinimumOSVersionKey (Platform))?.Value;
```

Comment:
rv can be null, does version.Parse have an issue with it? Would be nice to enable nullability in this code.

Reply:
https://github.com/xamarin/xamarin-macios/pull/14670